modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ProtobufSchemaCodegen.java

```diff
@@ -216,13 +215,12 @@ public String toOperationId(String operationId) {
     public Map<String, Object> postProcessModels(Map<String, Object> objs) {
         objs = postProcessModelsEnum(objs);
         List<Object> models = (List<Object>) objs.get("models");
-        // add x-index to properties
-        ProcessUtils.addIndexToProperties(models, 1);
 
         for (Object _mo : models) {
             Map<String, Object> mo = (Map<String, Object>) _mo;
             CodegenModel cm = (CodegenModel) mo.get("model");
 
+            int index = 1
```

Comment:
@k0ral overall it looks good 👍 but this line is missing the `;` at the end.

Reply:
Fixed, thanks !

Reply:
👍  Will merge after all the tests pass.